### PR TITLE
Feature/bird detail images

### PR DESCRIPTION
**Pull request by sydnerd**
# Pull Request Template
## Description

-Complete adding images to the bird detail page.
-Added to the image fetch to make it more dynamic 

## Type of change
- [ ] Bug fix 
- [X] New feature
- [ ] Refactoring

## Checklist:
- [X] My code follows the Turing's guidelines of this project
- [X] My changes generate no new errors/warnings

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const getBirds = (currentState) => {
   .then(response => response.json())
 }
 
-export const getImages = (birdName) => {
-  return fetch(`https://api.flickr.com/services/rest/?method=flickr.photos.search&api_key=50906f9c24e50a02f0d6034a3c6df6d7&per_page=1&format=json&nojsoncallback=1&text=${birdName}`)
+export const getImages = (birdName, num) => {
```
Comment (k-atwhite) [inline src/apiCalls.js:9]: great idea adding another parameter so we can DRY up this call!

File: src/components/BirdDetails/BirdDetails.js [hunk 2]
```diff
@@ -3,12 +3,43 @@ import binoculars from "../../binoculars.svg";
...
+  const [photos, setPhotos] = useState([]);
+
+  const createURL = (photo) => {
+    const photoUrl = 'https://live.staticflickr.com/' + photo.server + '/' +
+      photo.id + '_' + photo.secret + '_b.jpg';
+
+      return photoUrl;
+  }
+
+  // getImages(scientificName, 3)
```
Comment (LKessell) [inline src/components/BirdDetails/BirdDetails.js]: We should remove this comment :)

File: src/components/BirdDetails/BirdDetails.js [hunk 3]
```diff
@@ -3,12 +3,43 @@ import binoculars from "../../binoculars.svg";
...
+          return createURL(photo)
+        })
+        setPhotos(url)
+      }
+
+    })
 
   return (
     <div className="bird-detail">
+      <img src={photos[0]} width='300px'/>
```
Comment (k-atwhite) [inline src/components/BirdDetails/BirdDetails.js:32]: I'm very curious if there is a dryer way to do this

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const NavBar = () => {
     <nav>
       <NavLink to="/">Choose State</NavLink>
       <NavLink to="/birds">List of Birds</NavLink>
-      <NavLink to="/seen">My Birds</NavLink>
+      <NavLink to="/myBirds">My Birds</NavLink>
```
Comment (LKessell) [inline src/components/NavBar/NavBar.js:14]: Good catch!

**Review by k-atwhite — COMMENTED** [review]
Good job re-using that call

**Review by LKessell — CHANGES_REQUESTED** [review]
There's a straggler comment, but looks great otherwise!

**Review by LKessell — APPROVED** [review]

File: src/components/BirdDetails/BirdDetails.js (hunk 2 above)
Comment (sydnerd) [reply src/components/BirdDetails/BirdDetails.js]: good call!

**Pull request merged**